netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -177,13 +173,8 @@ private static Http2Headers convertMetadata(Metadata headers) {
 
     @Override
     public EventLoopGroup create() {
-      return new NioEventLoopGroup(0, new ExecutorServiceFactory() {
```

Comment:
ExecutorServiceFacotry and that particular constructor of NioEventLoopGroup only exist in 5.x